server/src/clients/s3storageClient.ts

```diff
@@ -116,6 +116,38 @@ export class S3StorageClient {
     }
   }
 
+  async deletePrefix(prefix: string): Promise<number> {
```

Comment:
```suggestion
  async hardDeletePrefix(prefix: string): Promise<number> {
```

--- just to keep in theme with the severity of calling this